tests/unit/parse-test.js

```diff
@@ -9,7 +9,7 @@ describe('Parse', () => {
     describe('literals', () => {
       it('handles language subtags', () => {
         let base = 'https://www.wikidata.org/wiki/Special:EntityData/Q2005.ttl'
-        let mimeType = 'text/turtle'
+        let mimeType = 'text/turtle; charset="UTF-8"'
```

Comment:
this test verifies that `;` and anything that follows is stripped and not considered.

`$rdf.parse` expects contentType to be a string of only the `type-name` and `subtype-name` for example `text/turtle` or `text/html` and so forth, with NO parameters,

Thus this test verifies that parameters, whatever they may be, are stripped and not considered